doc/3/controllers/collection/list/index.md

```diff
@@ -0,0 +1,61 @@
+---
+code: true
+type: page
+title: list
+description: Returns the collection list of an index
+---
+
+# list
+
+Returns the list of collections associated to a provided index.
+The returned list is sorted in alphanumerical order.
+
+<br/>
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> list(
+      final String index,
+      final ListOptions options) throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> list(
+      final String index) throws NotConnectedException, InternalException
+```
+
+
+| Arguments | Type                   | Description   |
+| --------- | ---------------------- | ------------- |
+| `index`   | <pre>String</pre>      | Index name    |
+| `options` | <pre>ListOptions</pre> | List options |
```

Comment:
I think we should get rid of these optional parameters: https://github.com/kuzzleio/kuzzle/pull/1592

If you don't, then the default value for the `size` parameter should be `null` from this SDK's point of view.

Reply:
I've didnt see that, thanks